src/sass/_common/common.scss

```diff
@@ -791,3 +791,19 @@ label + input {
         }
     }
 }
+
+.cashier-note {
+    background-color: $COLOR_LIGHT_GRAY;
+    color: $COLOR_BLUE;
+    display: inline-flex;
+    align-items: center;
+    margin: 1em 0;
+
+    &:before {
+        content: 'ⓘ';
```

Comment:
can you check the class we have for `notice-circle` and see how that's implemented? would it be better to reuse that instead of adding a second class for this info circle?